neo/io/nwbio.py

```diff
@@ -0,0 +1,810 @@
+"""
+NWBIO
+========
+
+IO class for reading data from a Neurodata Without Borders (NWB) dataset
+
+Documentation : https://neurodatawithoutborders.github.io
+Depends on: h5py, nwb, dateutil
+Supported: Read, Write
+Specification - https://github.com/NeurodataWithoutBorders/specification
+Python APIs - (1) https://github.com/AllenInstitute/nwb-api/tree/master/ainwb
+	          (2) https://github.com/AllenInstitute/AllenSDK/blob/master/allensdk/core/nwb_data_set.py
+              (3) https://github.com/NeurodataWithoutBorders/api-python
```

Comment:
PyNWB is not mentioned?

Reply:
Thanks for the pointer. You are right. PyNWB is missing. I add it.